proto/api.go

```diff
@@ -308,14 +307,139 @@ func (sr *ReverseScanResponse) Verify(req Request) error {
 	return nil
 }
 
+// GetInner returns the Request contained in the `oneof`.
+func (ru BatchRequest_RequestUnion) GetInner() Request {
```

Comment:
The whole `BatchRequest_RequestUnion_foo` set of types seems unnecessary on the part of gogoprotobuf. If the `ru.Request` was simply the type we placed in the batch (e.g. `GetRequest`) then the switch would be unnecessary and we could do what I described below (type assert `ru.Request.(Request)`). I think it was implemented this way so that `ru.Request` could enforce that only certain types be assigned. But that safety comes at a pretty steep cost.